src/org/thoughtcrime/securesms/ConversationListAdapter.java

```diff
@@ -51,8 +50,8 @@ class ConversationListAdapter extends RecyclerView.Adapter {
   private static final int MESSAGE_TYPE_THREAD         = 2;
   private static final int MESSAGE_TYPE_INBOX_ZERO     = 3;
 
-  private final WeakReference<Context>         context;
```

Comment:
why is the `WeakReference` to the application context removed? i am not sure, what it is for, iirc, this is to solve hard issues wrt garbage collection (to break cyclic references, even if rare in java)

so, i would not remove that without reasons and deep understanding.

Reply:
I removed it because it is never the case that context is null, it is always non-null, didn't realize it had other meaning, was thinking it was some superfluous wrapper similar to the "Stub" class is used in other places, will revert that